### PR TITLE
chore(ci): fast integer benchmarks to run on each push in main

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: tfhe/benches/integer/bench.rs [hunk 1]
```diff
@@ -35,9 +35,20 @@ impl Default for ParamsAndNumBlocksIter {
...
         };
 
+        let is_fast_bench = match env::var("__TFHE_RS_FAST_BENCH") {
+            Ok(val) => val.to_lowercase() == "true",
+            Err(_) => false,
+        };
+
+        let mut bit_sizes = vec![32];
+
         if is_multi_bit {
```
Comment (IceTDrinker) [inline tfhe/benches/integer/bench.rs:45]: I think we can better factorize this if you have some time otherwise should be good

**Review by IceTDrinker — APPROVED** [review]

**Comment by IceTDrinker** [comment]
@slab-ci cpu_wasm_test

File: tfhe/benches/integer/bench.rs (hunk 1 above)
Comment (soonum) [reply tfhe/benches/integer/bench.rs:45]: What do you mean by that ?

Comment (IceTDrinker) [reply tfhe/benches/integer/bench.rs:45]: let bit_sizes = if is_multi_bit {
...}

to have defaults per case

**Comment by soonum** [comment]
@slab-ci integer_bench

**Review by IceTDrinker — APPROVED** [review]
Thanks a lot easier to read for me 🙂

**Pull request merged**